frontend/src/app/pages/landing-page/landing-page/landing-page.component.ts

```diff
@@ -98,7 +97,7 @@ export class LandingPageComponent {
                 }
                 case 'partial': {
                     throw new Error(
-                        'Dieser Typ kann zur Zeit nicht importiert werden.'
+                        'Dieser Typ kann nur innerhalb einer Übung importiert werden.'
```

Comment:
The user does not see this error message, but only some complex stack trace (at least in development mode, may be different in prod):

![image](https://github.com/user-attachments/assets/ac76eef8-cd23-4292-99a3-ce436e9df271)

Reply:
I've redone this so that we post a message containing a more simple error, and no longer throw there. Throwing should be reserved for actual errors where a stack trace etc. is relevant.